core/server/models/plugins/pagination.js

```diff
@@ -90,6 +90,27 @@ paginationUtils = {
         }
 
         return pagination;
+    },
+
+    /**
+     *
+     * @param {Bookshelf.Model} model instance of Bookshelf model
+     * @param {string} propertyName property to be inspected and included in the relation
+     */
+    handleRelation: function handleRelation(model, propertyName) {
```

Comment:
This is the place where ordering populates `eagerLoad` relations array on the model. This allows for `eagerLoading` plugin to pick it up later and create needed join.